mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule UeberauthFacebook.Mixfile do
 
   defp deps do
     [{:ueberauth, "~> 0.4"},
-     {:oauth2, "0.6.0"},
+     {:oauth2, "~> 0.8.0"},
```

Comment:
I agree. I mean we should use `0.8` not `0.8.0` because looking at the source of `0.9.x` branch this probably won't be a breaking update